contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-ruby.mdx

```diff
@@ -85,4 +85,17 @@ is_my_flag_enabled = posthog.is_feature_enabled(
 
 import RubyOverrideServerProperties from './override-server-properties/ruby.mdx'
 
-<RubyOverrideServerProperties />
\ No newline at end of file
+<RubyOverrideServerProperties />
+
+### Feature flag timeouts
+
+You can use the `feature_flag_request_timeout_seconds` parameter on initialisation to set a configurable flag request timeout. This is useful so your requests don't block when PostHog's servers are too slow to respond. By default, this is set at 3 seconds.
+
+```ruby
+posthog = PostHog::Client.new({
+   api_key: "", # You can find this key on the /setup page in PostHog
+   personal_api_key: "", # Required for local feature flag evaluation
+   feature_flags_polling_interval: 10, # How often to poll for feature flags
```

Comment:
why do we include this?

Also,  I thought we only fetched feature flags when we call getFeatureFlags? (unless we're doign local eval)

Reply:
Right not necessary! Will remove 👍 

And that's correct we only fetch flags when we call `getFeatureFlags`, this polling is indeed for local evaluation, which can be very confusing here